File: Basic Programs/C C++/odd_Even.c
#include <stdio.h>

int main(){
    system("cls");
    int a;
    while(1){
    printf("Enter a Number: ");
    scanf("%d", &a);
    if(a&1 != 0){
        printf("Number is Odd\n");
    }else{
        printf("Number is Even\n");
    }
    }
}

File: Basic Programs/Java/odd-even.java
import java.util.Scanner;

public class jpe_2 {
    public static void main(String[] args) {
        
        Scanner sc = new Scanner(System.in);

        System.out.println("Please print a number");
        int i = sc.nextInt();

        if (i % 2 == 0) {
            System.out.println("Answer : " + i + " is an even number!");
        }

        else {
            System.out.println("Answer : " + i + " is an odd number!");
        }

        sc.close();

    }
}
